MANIFEST.in

```diff
@@ -1,11 +1,15 @@
+recursive-include docs *.rst *.html *.ico *.png *.py *.svg
+recursive-include examples *.py
 recursive-include falcon *.pyx
-recursive-include tests *.py
-recursive-include docs *.rst
+recursive-include tests *.py *.pyx
```

Comment:
is this used only for sdist? because I'm otherwise -1 on this since I think it would add docs, examples and tests packages to the installed python packages when installing from pypi a wheel

Reply:
Docs and tests were already inside the sdist, I only added the missing files.

Reply:
And no, none of this makes it into a wheel. Only package data, metadata and modules do.

Reply:
Thanks for checking 👍